documentation/CHANGELOG_EXAMPLE.md

```diff
@@ -0,0 +1,474 @@
+* Moved "basePath" /qod/v0 to "url"-property and introduced "apiroot" in definition of server  @maxl2287 in https://github.com/camaraproject/QualityOnDemand/pull/252
+* Added statusInfo 'DELETE_REQUESTED' for qosStatus 'UNAVAILABLE' and clarified notification events in documentation by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/258
+
+### Fixed
+
+* NA
+  
+### Removed
+
+* NA
+
+## New Contributors
+* @ravindrapalaskar17 made their first contribution in https://github.com/camaraproject/QualityOnDemand/pull/215
+* @rartych made their first contribution in https://github.com/camaraproject/QualityOnDemand/pull/227
+
+**Full Changelog**: https://github.com/camaraproject/QualityOnDemand/compare/v0.9.0...v0.10.0
+
+# r2.4 - M4 release candidate
+
+## Release Notes
+
+* quality-on-demand v0.10.0-rc.2
+* Commonalities v0.2.0
+* ICM v0.1.0
+* Backward compatible: N/A
+
+**This release contains the second and final (M4) release-candidate for the v0.10.0 of the Quality-On-Demand (QoD) API**
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/v0.10.0-rc2/code/API_definitions/qod-api.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/v0.10.0-rc2/code/API_definitions/qod-api.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QualityOnDemand/blob/v0.10.0-rc2/code/API_definitions/qod-api.yaml)
+
+## Changes for quality-on-demand v0.10.0-rc.2
+
+Changes compared to r2.3
+
+* Added a new error example for DurationOutOfRangeForQoSProfile by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/259
+* Moved "basePath" /qod/v0 to "url"-property and introduced "apiroot" in definition of server  @maxl2287 in https://github.com/camaraproject/QualityOnDemand/pull/252
+* Added a note to maxDuration parameter within qosProfile schema about the limit of 86400 seconds by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/256
+* Added statusInfo 'DELETE_REQUESTED' for qosStatus 'UNAVAILABLE' and clarified notification events in documentation by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/258:
+  * notifications will be sent for all changes of QosStatus, even if initiated by the client.
+  * what will happen when qosStatus changes from 'AVAILABLE' to 'UNAVAILABLE' due to 'NETWORK_TERMINATED'
+ 
+**Full Changelog**: https://github.com/camaraproject/QualityOnDemand/compare/v0.10.0-rc...v0.10.0-rc2
+
+# r2.3 - M3 release candidate
+
+## Release Notes
+
+* quality-on-demand v0.10.0-rc.1
+* Commonalities v0.2.0
+* ICM v0.1.0
+* Backward compatible: N/A
+
+**This release contains the first (M3) release-candidate for v0.10.0 of the Quality-On-Demand (QoD) API**
+
+- **This release contains significant changes compared to v0.9.0, and it is not backward compatible**
```

Comment:
## quality-on-demand v0.10.0-rc.1

** quality-on-demand v0.10.0-rc.1 is the first release candidate version for v0.10.0 of the Quality-On-Demand (QoD) API. The version contains significant changes compared to v0.9.0 and it is not backward compatible:**